test/middleware/request_validation_open_api_3_test.rb

```diff
@@ -256,7 +256,7 @@ def app
   it "rescues JSON errors" do
     @app = new_rack_app(schema: open_api_3_schema)
     header "Content-Type", "application/json"
-    post "/apps", "{x:y}"
+    post "/characters", "{x:y}"
```

Comment:
This path doesn't exist, it returns 200. To test JSON errors, I changed existing path. :)

Reply:
👍